plugin.xml

```diff
@@ -9,21 +9,24 @@ xmlns:android="http://schemas.android.com/apk/res/android"
     <keywords>cordova</keywords>
 
     <engines>
-        <engine name="cordova" version=">=9.0.0"/>
-        <engine name="cordova-android" version=">=8.0.0"/>
-        <engine name="cordova-ios" version=">=6.0.0"/>
+        <engine name="cordova" version=">=7.1.0"/>
+        <engine name="cordova-android" version=">=8.1.0"/>
+        <engine name="cordova-ios" version=">=5.1.1"/>
     </engines>
 
     <js-module name="FirebaseDynamicLinks" src="www/FirebaseDynamicLinks.js">
         <merges target="cordova.plugins.firebase.dynamiclinks" />
     </js-module>
 
-    <preference name="APP_DOMAIN_NAME" />
+    <preference name="APP_DOMAIN_NAME" default="firebase_domain_url_prefix"/>
     <preference name="APP_DOMAIN_PATH" default="/" />
 
-    <dependency id="cordova-plugin-firebase-analytics" version="~5.0.0"/>
+    <dependency id="cordova-plugin-firebase-analytics" url="https://github.com/OutSystems/cordova-plugin-firebase-analytics.git#outsystems"/>
```

Comment:
we should aim to use a fixed version of the plugin and not using the branch because any change on the analytics plugin may break this plugin

Reply:
At the time we did this PR, we did not have a fixed version of the analytics plugin yet. Now we do, so I will update this.